mono/profiler/log-args.c

```diff
@@ -347,4 +341,6 @@ usage (void)
 	mono_profiler_printf ("\treport               create a report instead of writing the raw data to a file");
 	mono_profiler_printf ("\tzip                  compress the output data");
 	mono_profiler_printf ("\tport=PORTNUM         use PORTNUM for the listening command server");
+
+	exit (code);
```

Comment:
How is this going to interact with us passing arguments for the old profiler API? We will need a period of transition where both command-line arguments will be in use, so I am not sure exiting when passing an unknown argument is going to help us in that.

Reply:
Good point, we should not exit on unknown arguments.

Reply:
Actually, it's fine: In the log profiler, we just print an error if we can't parse an argument; we don't invoke `usage ()`.